Fix minor issues

raymondbernard opened the pull request:
Updates and Fixes
1) Resolved JSON Encoding Issues: We've addressed encoding problems with JSON by enforcing utf-8 encoding.
2) Typo Corrections: Fixed a few typos to ensure accurate code and documentation.
3) Enhanced Error Handling: Improved error handling to provide more informative and helpful error messages.
3) Added Dependency: Included boto3 in requirements.txt to ensure seamless integration with AWS services.
4) Environment Variable Security: Added .env to .gitignore to protect sensitive environment variables from being committed to version control.
5) Google AI Studio Key: Added google_ai_studio key to your local .env copy for secure authentication.
6) JSON Data Handling: Ensured consistent use of dictionaries when working with JSON data to simplify data manipulation and parsing.

raymondbernard committed:
fixed typo

raymondbernard committed:
fixed typo

raymondbernard committed:
fixed typo

raymondbernard committed:
add GOOGLE_AI_STUDIO key

raymondbernard committed:
add encoding utf-8

raymondbernard committed:
add better error handling

raymondbernard committed:
handle cases where model not found in benchmark_questions

raymondbernard committed:
benchmark_questions as dic

raymondbernard committed:
ingnore .env

raymondbernard committed:
fixed typo

raymondbernard committed:
add boto3

JamesHuckle commented on .gitignore:
```diff
@@ -5,7 +5,7 @@ __pycache__/
 # C extensions
 *.so
-
+.env 
```
This can already be found on line 125 under environments (thankfully!) 
https://github.com/autogenai/easy-problems-that-llms-get-wrong/blob/41b335506f3f408f095482bbb569bd2d355a5a47/.gitignore#L125

raymondbernard replied:
Yes it's already in the .gitingore file.   I was a little overzealous.

raymondbernard committed:
Delete main.ipynb

Let's retry this commit. It seems like a lot of the notebook's runtime went into it.

JamesHuckle reviewed (approved):
Thank you!!
